python/CMakeLists.txt

```diff
@@ -71,7 +71,9 @@ else()
   install(TARGETS _pycudaq LIBRARY DESTINATION cudaq)
 endif()
 
-add_subdirectory(runtime/cudaq/domains/plugins)
+if (NOT CUDAQ_DISABLE_CPP_FRONTEND)
```

Comment:
I see the issue you mentioned about the chemistry package linking against the python version. What does that mean long term for including them in the wheels? Will they have to become their own package?

Reply:
Even if they are their own package, we should look into not linking against the python libraries. I would need to better understand where that need is coming from to define a plan.